templates/ci/go.yml.metadata

```diff
@@ -0,0 +1,3 @@
+{
+  "iconTaskName": "Go"
```

Comment:
This icon isn't 32x32 or square. The VSTS Build designer handles it okay, but we should probably replace it with a 32x32.